cms/envs/aws.py

```diff
@@ -190,6 +190,7 @@
 #theming start:
 PLATFORM_NAME = ENV_TOKENS.get('PLATFORM_NAME', 'edX')
 STUDIO_NAME = ENV_TOKENS.get('STUDIO_NAME', 'edX Studio')
+TENDER_DOMAIN = ENV_TOKENS.get('TENDER_DOMAIN', None)
```

Comment:
This should fall back to the value defined in `common.py` look at `LANGUAGE_CODE` for an example.